modules/apps/data-engine/data-engine-test/src/testFunctional/macros/DERenderer.macro

```diff
@@ -65,6 +65,21 @@ definition {
 		}
 	}
 
+	macro assertDataInLinktoPageField {
+		if (!(isSet(index))) {
+			var index = "1";
+		}
+
+		var selectedPage = selenium.getElementValue("xpath=(//label[normalize-space(text())='${webContentLinkToPageFieldLabel}']/following-sibling::div//div[contains(@class,'input-group')]//input[contains(@type,'text')])[${index}]");
+
+		if (contains("${selectedPage}", "${webContentLinkToPage}")) {
+			echo("The ${selectedPage} is shown in field.");
+		}
+		else {
+			TestUtils.fail(message = "The ${selectedPage} is not shown in field.");
```

Comment:
@guivilarim, why this is necessary? Can you please describe more about this code?